lib/Doctrine/ORM/Mapping/MappingException.php

```diff
@@ -820,4 +820,21 @@ public static function infiniteEmbeddableNesting($className, $propertyName)
             )
         );
     }
+
+    /**
+     * @param LibXMLError[] $errors
+     */
+    public static function fromLibXmlErrors(array $errors) : self
+    {
+        return new self(array_reduce(array_map(function (\LibXMLError $error) : string {
```

Comment:
First time ever playing with `array_reduce`, this is fun! But I should probably use implode instead, right?

Reply:
Yeah, that looks like it could perfectly be implemented with `implode(PHP_EOL, array_map(...);`.

Reply:
Will change